client/src/components/pipelines/browser/Folder.js

```diff
@@ -894,6 +911,73 @@ export default class Folder extends localization.LocalizedReactComponent {
     await this.createPipeline({name, description, repository, token});
   };
 
+  createVersionedStorage = async ({
+    name,
+    description,
+    repository,
+    pipelineType
+  }) => {
+    const hide = message.loading(`Creating versioned storage ${name}...`, 0);
+    await this.createPipelineRequest.send({
+      name: name,
+      description: description,
+      parentFolderId: this._currentFolder.folder.id,
+      repository: repository,
+      pipelineType
+    });
+    hide();
+    if (this.createPipelineRequest.error) {
+      message.error(this.createPipelineRequest.error, 5);
+    } else {
+      this.closeCreateVersionedStorageDialog();
+      this.props.pipelines.fetch();
+      await this.props.folder.fetch();
+      if (this.props.onReloadTree) {
+        this.props.onReloadTree(true);
+      }
+    }
+  };
+
+  onCreateVersionedStorage = async ({name, description, repository}) => {
+    const pipelineType = 'VERSIONED_STORAGE';
```

Comment:
@AleksandrGorodetskii we don't need `pipelineType` here: this method only checks the *repository* existence.